Integrations/SignalSciences/SignalSciences.yml

```diff
@@ -0,0 +1,980 @@
+commonfields:
+  id: Signal Sciences WAF
+  version: -1
+name: Signal Sciences WAF
+display: Signal Sciences WAF
+category: Network Security
+description: Protect your web application using Signal Sciences
+configuration:
+- display: Email - the email address connected to your account
+  name: Email
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Token - your authentication token
+  name: Token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Corporation Name
+  name: corpName
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Fetch Interval (the default time to fetch in the first fetch, given in minutes). Must be less than one week (10080 minutes)
+  name: fetch_interval
+  defaultvalue: "720"
+  type: 0
+  required: false
+- display: Sites to fetch from (List of site names, comma separated, without spaces).
+    If no sites are specified, events from all corp's sites will be fetched
+  name: sites_to_fetch
+  defaultvalue: ""
+  type: 12
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: sigsci-get-whitelist
+    arguments:
+    - name: siteName
+      required: true
+      description: The site that holds the whitelist you wish to get
+    outputs:
+    - contextPath: SigSciences.Whitelist.ID
+      description: ID for this particular entry.
+      type: string
+    - contextPath: SigSciences.Whitelist.Source
+      description: IP Address present in the whitelist.
+      type: string
+    - contextPath: SigSciences.Whitelist.ExpiryDate
+      description: Expiration Timestamp.
+      type: date
+    - contextPath: SigSciences.Whitelist.Note
+      description: Note associated with the tag.
+      type: string
+    - contextPath: SigSciences.Whitelist.CreatedDate
+      description: Created Date Timestamp
+      type: date
+    - contextPath: SigSciences.Whitelist.CreatedBy
+      description: User who added this source.
+      type: string
+    description: Fetch whitelist residing on Signal Sciences' Platform.
+  - name: sigsci-get-blacklist
+    arguments:
+    - name: siteName
+      required: true
+      description: The site that holds the blacklist you wish to get
+    outputs:
+    - contextPath: SigSciences.Blacklist.ID
+      description: ID for this particular entry.
+      type: string
+    - contextPath: SigSciences.Blacklist.Source
+      description: IP Address present in the blacklist.
+      type: string
+    - contextPath: SigSciences.Blacklist.ExpiryDate
+      description: Expiration Timestamp.
+      type: date
+    - contextPath: SigSciences.Blacklist.Note
```

Comment:
Blacklist\whitelist are for a Corp or for a site?

Reply:
for site, fixed